examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    """Plot connectome given a covariance matrix and atlas maps"""
+    imgs = image.iter_img(atlas_maps)
+    regions_coords = np.array(
+        [plotting.find_xyz_cut_coords(img) for img in imgs])
```

Comment:
> I think that a for loop here would be better than a list comprehension.

How so? Are you saying you want something like this:

```python
region_coords = []

for img in imgs:
    region_coords.append(plotting.find_xyz_cut_coords(img))

region_coords = np.array(region_coords)
```

> The name of the function is also misleading. We should have a function named find_poi

maybe in a different issue?